app/assets/stylesheets/custom.css.scss

```diff
@@ -130,3 +130,17 @@ li { list-style: none; }
 .values {
   color: $site-white;
 }
+
+.bw {
+  -webkit-filter: grayscale(100%);
+  -webkit-transition: all .5s ease;
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Properties should be ordered -webkit-filter, -moz-transition, -ms-transition, -o-transition, -webkit-transition, transition<br>Avoid vendor prefixes.